answer.txt

```diff
@@ -1,4 +1,14 @@
 •What is a gitignore file and what should it include for a Java project?
+Ans. gitignore is used to skip or ingore the files that are not required to be commited like server configuration files specific to local, log files etc.For Java project  it usually 
+includes .class(compiled class files), .log(log files), .ctxt(BlueJ files), .mtj.tmp/(J2ME files), pacakage files(.jar,.zip) etc.
+ 
```

Comment:
Please add a few more file types that we can add in .gitignore for java projects